.github/workflows/go.yml

```diff
@@ -37,5 +37,5 @@ jobs:
 
     - name: Test
       run: |
-        export GOPATH=$(pwd):$(go env GOPATH)/bin
+        export GOPATH=$(go env GOPATH)/bin
```

Comment:
**Callout:** No longer necessary to append current working directory when defining `GOPATH`. Adding it will cause the build to fail.